conventional.yaml

```diff
@@ -665,6 +665,8 @@ datastore:
         host: "localhost"
         # port is unnecessary for connections via unix domain socket:
```

Comment:
Can you change this comment to "port is unnecessary for connections to localhost"?

Reply:
I removed the comment and set the default port to 3306 instead. I think this is less confusing.